src/test/scala/org/ergoplatform/mining/ErgoMinerPropSpec.scala

```diff
@@ -109,17 +109,13 @@ class ErgoMinerPropSpec extends ErgoPropertyTest {
     }
 
     // transactions reach computation cost block limit
-    checkCollectTxs(100000L, Int.MaxValue, withTokens = false)
+    checkCollectTxs(100000L, Int.MaxValue)
 
     // transactions reach block size limit
-    checkCollectTxs(Long.MaxValue, 4096, withTokens = false)
-
-    // too many tokens in fees
```

Comment:
test with tokens is still useful here to ensure that miner collect correct transactions from mempool even if they have tokens